### ISSUE TITLE
How to do 1:1 face verification?

### ISSUE THREAD

**Issue by prnvjb** [wujiyang/Face_Pytorch#7]
Hi,
Thanks for this compilation.
I'm trying to do one on one face verification specifically based on mobilefacenet. How to compare distance between two feature vectors generated by model. I found loss function for mobilefacenet is arcface. If that is the similarity measure, how to compute arcface loss between two feature vectors.
Please share your thoughts.
Regards!

**Comment by wujiyang** [comment]
Arcface loss is just training loss function,  it helps to train discriminative feature. If you want to conduct face verification on two images, you can use cosine distance or euclidean distance to evaluate the two feature vectors.

**Comment by ZZH2950228** [comment]
do you have the 1:1 face verification code and can you share with me thank you ?

**Comment by ypw-lbj** [comment]
请问您使用预训练模型了吗？
Have you used the pre training model?
